src/utilities/validation.ts

```diff
@@ -255,4 +255,28 @@ export const isHandle: ValidationMethodFactory = property => value => {
   }
 
   return `${property} must only contain lowercase letters, numbers, and dashes`
-}
\ No newline at end of file
+}
+
+export const isLowerCase: ValidationMethodFactory = property => value => {
+  if (isNullish(value) || isEmptyString(value)) {
+    return true
+  }
+
+  if (typeof value === 'string' && value === value.toLowerCase()) {
+    return true
+  }
+
+  return `${property} must be lowercase`
+}
+
+export const isUpperCase: ValidationMethodFactory = property => value => {
+  if (isNullish(value) || isEmptyString(value)) {
+    return true
+  }
+
+  if (typeof value === 'string' && value === value.toUpperCase()) {
+    return true
+  }
+
+  return `${property} must be uppercase`
+}
```

Comment:
There's a new type for validation rules for the compositions. `ValidationRule` is the type. I'm actually presently surprised that we can use the existing factory methods with the new compositions. For some reason I had in my mind that they wouldn't work.

Reply:
Using the ValidationRule type enabled you to do this
```
useValidation(value, 'name', [isRequired, isLowerCase])
```

Rather than
```
useValidateion(value, [isRequired('name'), isLowerCase('name'])
```

Reply:
Ah I can update these - I didn't realize I was using the old ones. These work perfectly though, but they're certainly verbose

Reply:
lol yeah they are really verbose. Another side effect of trying to fit things into vee-validate. But the compositions tried to make that better. You can also create type safe validation rules like `ValidationRule<string>`.

Reply:
I ended up removing these for now - can add them again later but felt this was getting too out of scope